app/lib/mock_router_api.rb

```diff
@@ -0,0 +1,35 @@
+# When DISABLE_ROUTER_API is set to true, the application will use this
+# class instead of GdsApi::Router
+# This is to allow dual-running of both MongoDB and PostgreSQL Content Stores
+# for a transition period during the migration to PostgreSQL
+class MockRouterApi
+  def delete_route(*args)
+    log("delete_route", args)
+  end
+
+  def add_backend(*args)
+    log "add_backend", args
+  end
+
+  def add_redirect_route(*args)
+    log "add_redirect_route", args
+  end
+
+  def add_gone_route(*args)
+    log "add_gone_route", args
+  end
+
+  def add_route(*args)
+    log "add_route", args
+  end
+
+  def commit_routes(*args)
+    log "commit_routes", args
+  end
+
+private
+
+  def log(method, args)
+    logger.debug "Mocked call to router_api: #{method}(#{args})"
```

Comment:
Should we be explicit and use Rails.logger - I hadn't actually realised that logger was a global method (does ActiveSupport patch it into Object?)

Reply:
Also, just wanted to sense choice of debug level, since that'll only run in dev and test environments, not production. Is that correct that you wouldn't want this to log when run in any of the GOV.UK environments? (I don't have an opinion on which you choose)

Reply:
Yeah, that's a fair point - `info` might be better while we're testing on integration and staging. We can take a judgement call before rolling out to production as to whether we want to adjust it back down again then.

Reply:
done